src/components/Envelope.vue

```diff
@@ -190,7 +190,8 @@ export default {
 	},
 	computed: {
 		account() {
-			return this.$store.getters.getAccount(this.data.accountId)
+			const accountId = this.data.accountId ? this.data.accountId : this.mailbox.accountId
```

Comment:
```suggestion
			const accountId = this.data.accountId ?? this.mailbox.accountId
```

or we just always take the ``this.mailbox.accountId``? :thinking:

Reply:
>or we just always take the this.mailbox.accountId?

In such case, I would also change the assignment at line 4

Reply:
In the meantime, I've applied the change you've requested